docs/notebooks/Supervised_WMD.ipynb

```diff
@@ -0,0 +1,1441 @@
+{
```

Comment:
<p>since this is copypasted from (https://ott-jax.readthedocs.io/en/latest/notebooks/One_Sinkhorn.html) you can remove this entire section and refer to the colab in the doc, and start from your contribution</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>